### ISSUE TITLE
Some EXDATE are not excluded as expected

### ISSUE THREAD

**Issue by britweb-tim** [u01jmg3/ics-parser#240]
- PHP Version: `7.3.8`
- PHP date.timezone: `UTC`
- ICS Parser Version: `2.1.15` (dev-master)
- Linux

### Description of the Issue:
EXDATES are not excluded correctly from g-suite calendar iCal feed.

### Steps to Reproduce:

Using the following initialisation for ICal
```
$ical = new ICal\ICal('https://calendar.google.com/...../basic.ics', [
        'defaultSpan'                 => 2,     // Default value
        'defaultTimeZone'             => 'UTC',
        'defaultWeekStart'            => 'MO',  // Default value
        'disableCharacterReplacement' => false, // Default value
        'filterDaysAfter'             => null,  // Default value
        'filterDaysBefore'            => null,  // Default value
        'skipRecurrence'              => false, // Default value
        'useTimeZoneWithRRules'       => false, // Default value
    ]);
```
The EXDATES from the following ICAL segment are not removed as expected. 

In particular the final date `Europe/London:20190911T095000` is not excluded.

However `Europe/London:20190925T095000` does appear to be excluded correctly.

```
BEGIN:VEVENT
DTSTART;TZID=Europe/London:20190911T095000
DTEND;TZID=Europe/London:20190911T110500
RRULE:FREQ=WEEKLY;WKST=SU;UNTIL=20191027T235959Z;BYDAY=WE
EXDATE;TZID=Europe/London:20191023T095000
EXDATE;TZID=Europe/London:20191009T095000
EXDATE;TZID=Europe/London:20190925T095000
EXDATE;TZID=Europe/London:20190911T095000
DTSTAMP:20190910T100203Z
UID:4ta52ghl3dq6rku8mh4btuicib@google.com
CREATED:20190822T150032Z
DESCRIPTION:
LAST-MODIFIED:20190902T190633Z
LOCATION:
SEQUENCE:2
STATUS:CONFIRMED
SUMMARY:Members Swim
TRANSP:OPAQUE
END:VEVENT
```

**Comment by thisanimus** [comment]
I'm experiencing the same behavior.  For me it's removing all but the last exdate.  It looks like if DTSTART is the same day as the first EXDATE, it does not remove it.

**Comment by u01jmg3** [comment]
@s0600204: can you help?

**Comment by u01jmg3** [comment]
Fixed by #246
